payment-delegation-db-relayer/nodejs/README.md

```diff
@@ -0,0 +1,38 @@
+# Payment Delegation Database
+
+<!-- TODO Add link to docs once it's published -->
+
+This directory contains an example usage of the Payment Delegation Database to register a new payer wallet, and add users as payees. There is a corresponding Lit docs page that covers the code in this example which is available here.
```

Comment:
Hyperlink?

Reply:
Added, link will just 404 until the docs PR is merged and published